File: Data Structures & Algorithms/design-hashmap/submission-0.java
class MyHashMap {
    int[] keyArray;
    public MyHashMap() {
        keyArray = new int[1000001];
        for(int i = 0; i < keyArray.length; i++){
            keyArray[i] = -1;
        }
    }
    
    public void put(int key, int value) {
        keyArray[key] = value;
    }
    
    public int get(int key) {
        int val = keyArray[key];
        if(val == -1){
            return -1;
        }
        return keyArray[key];
    }
    
    public void remove(int key) {
        keyArray[key] = -1;
    }
}

/**
 * Your MyHashMap object will be instantiated and called as such:
 * MyHashMap obj = new MyHashMap();
 * obj.put(key,value);
 * int param_2 = obj.get(key);
 * obj.remove(key);
 */

File: Data Structures & Algorithms/insert-delete-getrandom-o1/submission-0.py
class RandomizedSet:

    def __init__(self):
        self.vals = set()


    def insert(self, val: int) -> bool:
        if val in self.vals:
            return False
        self.vals.add(val)
        return True

    def remove(self, val: int) -> bool:
        if val not in self.vals:
            return False
        self.vals.remove(val)
        return True

    def getRandom(self) -> int:
        newList = list(self.vals)
        idx = random.randint(0, len(self.vals) - 1)
        return newList[idx]


# Your RandomizedSet object will be instantiated and called as such:
# obj = RandomizedSet()
# param_1 = obj.insert(val)
# param_2 = obj.remove(val)
# param_3 = obj.getRandom()